apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -301,7 +302,9 @@ namespace MWWorld
         MWWorld::Ptr ptr = ref.getPtr();
 
         osg::Vec4 lightDiffuseColor = getMagicBoltLightDiffuseColor(state.mEffects);
-        createModel(state, ptr.getClass().getModel(ptr), pos, orient, true, true, lightDiffuseColor, texture);
+
+        const std::string model = ptr.getClass().getModel(ptr);
```

Comment:
This change and the identical added line below look redundant, and so do the formatting changes.